src/unitycatalog/ai/databricks.py

```diff
@@ -106,11 +106,32 @@ class DatabricksFunctionClient(BaseFunctionClient):
     Databricks UC function calling client
     """
 
-    def __init__(self, client: Optional["WorkspaceClient"] = None, **kwargs: Any) -> None:
+    def __init__(
+        self,
+        client: Optional["WorkspaceClient"] = None,
+        *,
+        warehouse_id: Optional[str] = None,
+        cluster_id: Optional[str] = None,
+        profile: Optional[str] = None,
+        **kwargs: Any,
+    ) -> None:
+        """
+        Databricks UC function calling client.
+
+        Args:
+            client: The databricks workspace client. If it's None, a default databricks workspace client
+                is generated based on the configuration. Defaults to None.
+            warehouse_id (str, optional): The warehouse id to use for executing functions. This field is
```

Comment:
```suggestion
            warehouse_id: The warehouse id to use for executing functions. This field is
```

because we have type hints.